src/plugins/console/public/src/es.js

```diff
@@ -32,7 +32,7 @@ module.exports.send = function (method, path, data, server, disable_auth_alert)
   var settings = require("./settings");
 
   var options = {
-    url: '../api/sense/proxy?uri=' + encodeURIComponent(path),
+    url: '../api/console/proxy?uri=' + encodeURIComponent(path),
```

Comment:
Is this basepath compatible?  The same would apply to the relative path in the next file.

Not an issue with this PR, though.